full-stack-tests/core/src/frontend/standalone/ModelState.test.ts

```diff
@@ -126,4 +126,50 @@ describe("ModelState", () => {
     assert.isTrue(range.low.isAlmostEqual({ x: 288874.1174466432, y: 3803761.1888925503, z: -0.0005 }));
     assert.isTrue(range.high.isAlmostEqual({ x: 289160.8417204395, y: 3803959.118535, z: 0.0005 }));
   });
+
+  it("view thumbnails", async () => {
+    let thumbnail = await imodel3.views.getThumbnail("0x34");
+    assert.equal(thumbnail.format, "png", "thumbnail format");
+    assert.equal(thumbnail.height, 768, "thumbnail height");
+    assert.equal(thumbnail.width, 768, "thumbnail width");
+    assert.equal(thumbnail.image.length, 19086, "thumbnail length");
+    const image = thumbnail.image;
+    assert.equal(image[0], 0x89);
+    assert.equal(image[1], 0x50);
+    assert.equal(image[2], 0x4E);
+    assert.equal(image[3], 0x47);
+    assert.equal(image[4], 0x0D);
+    assert.equal(image[5], 0x0A);
+    assert.equal(image[6], 0x1A);
+    assert.equal(image[7], 0x0A);
+
+    thumbnail = await imodel2.views.getThumbnail("0x24");
+    assert.equal(thumbnail.format, "jpeg");
+    assert.equal(thumbnail.height, 768);
+    assert.equal(thumbnail.width, 768);
+    assert.equal(thumbnail.image.length, 18062);
+    assert.equal(thumbnail.image[3], 224);
+    assert.equal(thumbnail.image[18061], 217);
+
+    // thumbnail.format = "png";
+    // thumbnail.height = 100;
+    // thumbnail.width = 200;
+    // thumbnail.image = new Uint8Array(301); // try with odd number of bytes
+    // thumbnail.image.fill(33);
+
+    // await imodel2.views.saveThumbnail("0x24", thumbnail);
```

Comment:
What's up with the commented-out code?

Reply:
This PR is mostly reverting [this PR](https://github.com/iTwin/itwinjs-core/pull/3138/files) so the commented out code was copied from the test that was removed when this function was first deprecated. If it was commented out before, probably not worth keeping so I'll remove this.

Reply:
Eh, `saveThumbnail` still exists and is *not* deprecated - it would be nice if it was tested.
It seems kind of odd to allow people to save thumbnails but deprecate the method that allows them to subsequently retrieve them... @calebmshafer?

Reply:
Ah - I'm looking at the backend saveThumbnail method, which is fine (as is the backend getThumbnail).
The frontend test can't save a thumbnail. Deleting that part is good.